src/applications/vaos/utils/formatters.js

```diff
@@ -1,3 +1,6 @@
+import { first, includes, last, replace, split, toLower } from 'lodash';
```

Comment:
It is recommended to use JS built-in functions where appropriate:

- [`.toLowerCase()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase)
- [`.split()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split)
- [`.replace()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace)
- [`.includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) (the string version)

Reply:
This is code that was copied over from a different app, per VSP recommendation.